meta-genivi-dev/recipes-sota/python-storm/python-storm_0.20.bb

```diff
@@ -0,0 +1,37 @@
+SUMMARY = "Storm is an object-relational mapper (ORM) for Python developed at Canonical"
+HOMEPAGE = "https://pypi.python.org/pypi/storm"
+LICENSE = "LGPL-2.1"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e91355d8a6f8bd8f7c699d62863c7303"
+
+SRC_URI = "https://launchpad.net/storm/trunk/0.20/+download/storm-0.20.tar.gz \
+  "
+SRC_URI[md5sum] = "8628503141f0f06c0749d607ac09b9c7"
+
+S = "${WORKDIR}/storm-0.20"
+
+inherit setuptools
+
+DEPENDS = "python "
+RDEPENDS_${PN} += " python-setuptools \
+                    python-pkgutil \
+                    python-modules \
+                  "
+
+do_compile_prepend() {
+    BUILD_SYS=${BUILD_SYS} HOST_SYS=${HOST_SYS} \
+    ${STAGING_BINDIR_NATIVE}/python-native/python setup.py install || \
+    true
+}
+
+# need to export these variables for python-config to work
+export PYTHONPATH
+export BUILD_SYS
+export HOST_SYS
+export STAGING_INCDIR
+export STAGING_LIBDIR
+
+FILES_${PN} = " \
+  /usr/lib/python2.7/site-packages \
```

Comment:
use 4 spaces.
/usr/lib -> ${libdir}
python2.7 -> ${PYTHON_DIR} right? please check this again.